lib/BUILD

```diff
@@ -2,7 +2,14 @@ package(default_visibility = ["//visibility:public"])
 
 ##### Aggregate libraries #####
 
+# Libraries of the following form:
```

Comment:
Perhaps move this comment above `##### Aggregate libraries #####` as this tag is also used with other libraries?

Reply:
Good point - done.